libs/vre/shared/app-api/src/lib/services/admin/project-api.service.ts

```diff
@@ -19,7 +21,8 @@ export type ProjectIdentifier = 'iri' | 'shortname' | 'shortcode';
 export class ProjectApiService extends BaseApi {
   constructor(
     private _http: HttpClient,
-    private _appConfig: AppConfigService
+    private _appConfig: AppConfigService,
+    private adminProjectsApiService: AdminProjectsApiService
```

Comment:
don't add this here, we should have openApi services and app-api separated, as we are going to delete app-api anyway